flow360/component/flow360_params/flow360_params.py

```diff
@@ -1264,6 +1268,7 @@ class Flow360Params(Flow360BaseModel):
     monitor_output: Optional[MonitorOutput] = pd.Field(alias="monitorOutput")
     volume_zones: Optional[VolumeZones] = pd.Field(alias="volumeZones")
     aeroacoustic_output: Optional[AeroacousticOutput] = pd.Field(alias="aeroacousticOutput")
+    unit_defaults: Optional[UnitDefaults] = pd.Field(alias="unitDefaults")
```

Comment:
create `ROOT/tools/integrations/unit_systems_defaults.py` script that will export these